client/knowledge_base_search/src/containers/SearchContainer/SearchContainer.js

```diff
@@ -42,84 +45,108 @@ class SearchContainer extends Component {
     return `View all ${numPresetQueries.toLocaleString()}+ questions`;
   }
 
-  render() {
+  handleOnQuestionTypeSelect = (e) => {
+    this.setState({ selectedQuestionType: e.target.value });
+  }
+
+  renderQuestionType() {
     const {
-      errorMessage,
       isFetchingQuestions,
-      isFetchingResults,
-      presetQueries,
+      errorMessage,
       offset,
       onOffsetUpdate,
       onQuestionClick,
+      presetQueries,
+      isFetchingResults,
       onViewAllClick
     } = this.props;
 
+    switch (this.state.selectedQuestionType) {
+      case QuestionTypeSelect.questionTypes.PRESET.value:
+        return (
+          <div>
+          {
+            isFetchingQuestions
+              ? (
+                  <div key='loader' className='_container _container_large _container-center'>
```

Comment:
The underscore prefixes on the classes seems a little strange to me. What's up with that?

Reply:
i agree.